src/Model/Shipment.php

```diff
@@ -608,7 +608,10 @@ public function getItems(): array
      */
     public function setItems(array $items): self
     {
-        $this->items = $items;
+        $this->items = [];
+        foreach ($items as $item) {
```

Comment:
This should be for setting items directly, like if another logic should dictate how items are added.